core/driver/sources/src/main/resources/reference.conf

```diff
@@ -125,5 +125,13 @@ probe {
     // treat all projects/repositories under the `trusted` directory as "safe" and will run the code without alerts.
     // By default, `trusted` is resolved as the root directory: "/".
     # trusted = "/"
+    //
+    // `diagnostics` is a directory where diagnostics data, like logs from the intelliJ instance, can be copied into
+    // before cleanup of the IntelliJ instance. Specify proper path if you want to have access to these
+    // files after tests finish (could be useful when working with CI pipelines). By default, `diagnostics` directory
+    // is NOT created and such data is deleted from the disk with all other files from the IntelliJ instance directory.
+    // If you use this parameter, the `diagnostics` directory will be created and it will NOT be removed/cleared.
+    // Example path below:
+    # diagnostics = "/tmp/ide-probe/diagnostics-data"
```

Comment:
I'm uncertain about this `diagnostics` name... how about `exportLogsTo` or `logExport`? WDYT Łukaszes?

Reply:
I'd just like to differentiate the paths passed to IntelliJ as configs (all others) from this one, which isn't an IntelliJ config path per se.

Reply:
Good idea! Renamed to logExport.